.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+#### Ticket
+[TICKET_NUMBER](https://vimean.atlassian.net/browse/TICKET_NUMBER)
```

Comment:
Non-vimeans will not have a ticket number, consider adding a note that this field is optional or for Vimeans-only